app/controllers/taxon_changes_controller.rb

```diff
@@ -399,6 +400,14 @@ def load_taxon_change
       ).first
   end
 
+  def stub_taxon_split
+    input_taxon = Taxon.where( id: params[:inputTaxonId].to_i ).first
+    output_taxa = Taxon.where( id: params[:outputIds].map{|i| i.to_i} )
+    @taxon_change = TaxonSplit.new
+    @taxon_change.taxon = input_taxon
+    @taxon_change.new_taxa << output_taxa
```

Comment:
Doesn't using `<<` actually save the `@taxon_change` here and save the `TaxonChangeTaxa` records? I would have thought you would need to do something like `output_taxa.each { |t| @taxon_change.taxon_change_taxa.build( taxon: t )` to make a record without saving it.